modules/virt-viewing-network-state-of-node-console.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-observing-node-network-state.adoc
+
+:_content-type: PROCEDURE
+[id="virt-viewing-network-state-of-node-console_{context}"]
+= Viewing the network state of a node from the web console
+
+As an administrator, you can use the {product-title} web console to observe `NodeNetworkState` resources and network interfaces, and access network details.
+
+.Procedure
+. Navigate to *Networking* → *NodeNetworkState*.
++
+In the *NodeNetworkState* page, you can view the list of `NodeNetworkState` resources, and the interfaces that are created on the nodes.
+. Click on the `NodeNetworkState` resource name listed in the *Name* column to access the detailed information about NNS.
+. Click on the *>* icon to expand and view the *Network Details* section for the `NodeNetworkState` resource. Alternatively, you can click on each interface type under the *Network interface* column to view the network details.
```

Comment:
* IBM SG says to omit the "on" and just use "click the ..."
* Generally, it's better to put the goal of the action first so that the user knows if they care about the step at all. So, for example: 
```
To expand and view the *Network Details* section for the `NodeNetworkState` resource, click the *>* icon.
```